go-controller/pkg/util/net.go

```diff
@@ -140,13 +140,17 @@ func JoinIPs(ips []net.IP, sep string) string {
 }
 
 // JoinIPNets joins the string forms of an array of *net.IPNet, as with strings.Join
-func JoinIPNets(ipnets []*net.IPNet, sep string) string {
+func JoinIPNets(ipnets []*net.IPNet, sep string, ipOnly bool) string {
```

Comment:
boolean flags are tricky... I would have expected `true` to mean "show the subnet length" and `false` to mean "skip it".

Maybe just have a separate `JoinIPNetIPs()`?

Reply:
That's funny. I originally wrote it that way (separate function), then consolidated when self reviewing, thinking you would want less code. Will break back into separate function.